base/compose.yaml

```diff
@@ -31,6 +32,7 @@ services:
 
     ports:
       - "{API_PORT}:{NGINX_PORT}"
+      - "{DOCS_PORT}:{DOCS_PORT}"
```

Comment:
I'd like it if you could find some way to make the ports not collide by default when DOCS_PORT isn't set. Maybe default it to `"1{API_PORT}"` or something